types/user/user.ts

```diff
@@ -0,0 +1,11 @@
+export interface User {
+  user_id: string;
+  username: string;
+  first_name: string;
+  last_name: string;
+  email: string;
+  password: string;
+  created_at: Date;
+  updated_at: Date;
+  documents: AnnoteDocument[];
```

Comment:
Since there is a one-to-many relationship with `Users` and  `AnnoteDocuments`, the `AnnoteDocuments` table will have its `user_id` marked as a foreign key.

I think we should delete this from the interface and from the schema?

Reply:
Great point! Will remove this